frontend/src/features/public-form/mutations.ts

```diff
@@ -80,6 +81,12 @@ export const usePublicFormMutations = (
     },
   )
 
+  const submitStorageModeFormV2Mutation = useMutation(
```

Comment:
likewise, suggest to also have a fetch fallback

Reply:
fetch version created in [d6e9db3](https://github.com/opengovsg/FormSG/pull/6665/commits/d6e9db359c1e494a792b28388408643b8bf8eb45)